zad4/db/migrate/20181123195831_lets_see_how_migrations_work.rb

```diff
@@ -0,0 +1,5 @@
+class LetsSeeHowMigrationsWork < ActiveRecord::Migration[5.2]
+  def change
+    change_column_default(:players, :captain, :false)
```

Comment:
Ta migracja popsuła CI troszkę seedy, które bazowały na deafualtowej wartości :) Co tu się stało? Jako default został ustawiony symbol a nie boolean, `:false` zamiast `false` :D Railsy chcąc być bardzo mądre postanowiły Ci zamienić symbol na boolean, a jak wiadomo, jak coś nie jest nilem, to zamieniane jest na `true` ;)